README.md

```diff
@@ -130,28 +147,40 @@ evince <filename>.eps
 ```
 Replace `<filename>` with the name of the file (`map` or `cartogram`) you wish to open.
 
-For our 2016 US election example, `map.eps` should look like:
+For our example, `map.eps` should look as follows:
 
 <p align="center">
-<img src ="sample_images/US_map.png" width="65%">
+<img src ="sample_images/usa_map.png" width="65%">
 </p>
 
 `cartogram.eps` should look like:
 
 <p align="center">
-<img src ="sample_images/US_election_2016_cartogram.png" width="65%">
+<img src ="sample_images/usa_cartogram_electors.png" width="65%">
 </p>
 
-## Troubleshooting
-### Error: Don't understand 'm' flag!
+### Options
 
-If you see the following line in your output, you may want to take a look at <a href="https://github.com/dmlc/xgboost/issues/1945" target="\_blank">this solution</a>. Once you have implemented the solution from the link, follow the standard build instructions above.
-```
-FATAL:/opt/local/bin/../libexec/as/x86_64/as: I don't understand 'm' flag!
-```
+**-e**<br/>
```

Comment:
Please explain this option. Also, should the default be to generate the EPS, and there could be an option to not generate it?

Reply:
I've added more explanation of this option in 4287467. I'd rather not change the default behavior of the application, because this would mean updating the go-cart web application.